susemanager/susemanager.spec

```diff
@@ -62,7 +62,7 @@ Name:           susemanager
 Version:        5.1.9
 Release:        0
 Summary:        %{productprettyname} specific scripts
-License:        GPL-2.0-only
+License:        GPL-2.0-only AND Apache-2.0
```

Comment:
This is what we were told we are not allowed to do in https://github.com/uyuni-project/uyuni/pull/10032. Is that not the case anymore?

Reply:
Hmmm, I was not aware of this. I also wrongly assumed `GPL-2.0-only` means just that is not `GPL-2.0-or-later`.

In this case, we cannot have `mgr-salt-ssh` in a package that is licensed as `GPL-2.0-only`, so either move it to a different package or we relicense `susemanager-tools` package. I'll have some thoughts on this.

Thanks for the review!

Reply:
I probably should have linked to the [particular comment](https://github.com/uyuni-project/uyuni/pull/10032#discussion_r2007549311) I was referring to.

>  I also wrongly assumed GPL-2.0-only means just that is not GPL-2.0-or-later.

That's correct, as far as I understand. The problem is that the FSF says that Apache-2.0 is incompatible with GPL-2.0. In #10032 I was told I can't have this incompatible license tag in the spec file. I don't know if that's true or not, just wanted to make you aware of the feedback I got previously.